README.md

```diff
@@ -54,8 +54,10 @@ Many of our icons came from [the Noun Project][nouns]:
 * Weekend: "[Sunglasses][sunglasses]" By NAS, from the Noun Project.
 * Next Month: "[Calendar][calendar]" By artworkbean, from the Noun Project. (Modified)
 * Pick a Date: "[Calendar][calendar]" By artworkbean, from the Noun Project.
+* Undo: "[Undo][]" By H Alberto Gongora, from the Noun Project.
 
 
+[undo]: https://thenounproject.com/search/?q=undo&i=716798
```

Comment:
We should leave these in alphabetical order, since I went to all that effort in the first place…  😉